libs/community/langchain_community/utilities/pebblo.py

```diff
@@ -188,15 +206,21 @@ def get_loader_full_path(loader: BaseLoader) -> str:
                 location = f"gc://{loader.bucket}/{loader.blob}"
             elif isinstance(loader, S3FileLoader):
                 location = f"s3://{loader.bucket}/{loader.key}"
+        elif "source" in loader_dict:
+            location = loader_dict.get("source", "-")
+            if "channel" in loader_dict:
+                location = location + "/" + loader_dict.get("channel", "-")
         elif "path" in loader_dict:
-            location = loader_dict["path"]
+            location = loader_dict.get("path", "-")
```

Comment:
Why is `-` a default type? Why are we using `get` if the value has already been confirmed to be inside loader_dict?

Reply:
Thanks for your input @eyurtsev! 
The redundant use of - has been addressed, and adjustments were made to resolve linting errors. 
Changes have been reverted to enhance code quality and readability.
Please review